docs/en/operation/User-CLI.md

```diff
@@ -597,6 +607,16 @@ The `--replication` flag can be used to load the data into multiple workers.
 $ ./bin/alluxio fs distributedLoad --replication 2 /data/today
 ```
 
+### distributedMv
+
+The `distributedMv` command moves a file or directory in the Alluxio file system distributed across workers.
```

Comment:
If it's just one file is it still distributed somehow? Or does it just fall back to using a regular move?

Reply:
I should say it uses the JobService here. (No fallback)